nvflare/client/api.py

```diff
@@ -45,10 +46,11 @@ def init(rank: Optional[str] = None):
     api_type_name = os.environ.get(CLIENT_API_TYPE_KEY, ClientAPIType.IN_PROCESS_API.value)
     api_type = ClientAPIType(api_type_name)
     global client_api
-    if api_type == ClientAPIType.IN_PROCESS_API:
-        client_api = data_bus.get_data(CLIENT_API_KEY)
-    else:
-        client_api = ExProcessClientAPI()
+    if client_api is None:
+        if api_type == ClientAPIType.IN_PROCESS_API:
+            client_api = data_bus.get_data(CLIENT_API_KEY)
+        else:
+            client_api = ExProcessClientAPI()
 
     client_api.init(rank=rank)
```

Comment:
Shouldn't you move client_api.init() to the inside of the "if client_api is None", so it will be called only once?

Reply:
thanks, changed as suggested